autoload/health/check.lua

```diff
@@ -0,0 +1,14 @@
+function nvim_lsp_check_health()
+	local configs = require 'nvim_lsp/configs'
+
+	for _, top_level_config in pairs(configs) do
+		-- the folder needs to exist
+		local new_config = top_level_config.make_config(".")
+		vim.fn['health#report_info']("Checking "..new_config.name)
```

Comment:
this is noise. instead, show a success message for non-error cases.

Reply:
It's difficult to guarantee the success with basic or even more exhaustive checks. 
I just wanted the user to see their config was at least found. So maybe instead: `Found config for X` or `No obvious error for X`.

Reply:
no need to overthink it. "success" is the opposite of "error".

Reply:
I chose `configuration checked`